pytorch_lightning/callbacks/lr_logger.py

```diff
@@ -38,9 +38,10 @@ def configure_optimizer(self):
                             'name': 'my_logging_name'}
             return [optimizer], [lr_scheduler]
     """
-    def __init__(self):
+    def __init__(self, global_step=True):
```

Comment:
can you add some type annotations and docstrings here?

Reply:
yes, I will have a try